parsers/cxx/cxx_parser_function.c

```diff
@@ -1494,6 +1494,7 @@ int cxxParserEmitFunctionTags(
 		// so "sane" such declarations are short and usually don't have meaningful tags inside.
 
 	} else {
+		pSavedScope = NULL;
```

Comment:
I feel initializing pSavedScope to NULL at the line where pSavedScope is defined is a bit better than assigning NULL here. I mean:
```
CXXToken * pSavedScope = NULL;
```

Reply:
Technically that's slightly less efficient as with the optimization turned off in the positive case the assignment will be done twice. With optimization turned on it probably won't matter (?).